plots/measure.js

```diff
@@ -0,0 +1,226 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const path = require('path');
+const parseURL = require('url').parse;
+
+const mkdirp = require('mkdirp');
+
+const constants = require('./constants.js');
+const utils = require('./utils.js');
+
+const lighthouse = require('../lighthouse-core/index.js');
+const ChromeLauncher = require('../lighthouse-cli/chrome-launcher.js').ChromeLauncher;
+const Printer = require('../lighthouse-cli/printer');
+const assetSaver = require('../lighthouse-core/lib/asset-saver.js');
+
+const NUMBER_OF_RUNS = 20;
+
+const URLS = [
+  // Flagship sites
+  'https://nytimes.com',
```

Comment:
might also be worth throwing in some of [these](https://docs.google.com/document/d/1X3cuSmTKcnUhoeo3-5Vr6SLAwLwCUgr9OQtSKKKCK_o/edit)